File: PUSGSProjekat2/ProjekatPUSGS/ProjekatPUSGS/Models/BrzaRezervacijaVozila.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ProjekatPUSGS.Models
{
    public class BrzaRezervacijaVozila
    {
        [Key]
        public int Id { get; set; }

        public int IdServisa { get; set; }

        public int IdVozila { get; set; }

        public double NovaCena { get; set; }

        public double PocetnaCena { get; set; }

        public double Popust { get; set; }

        public bool Zavrseno { get; set; }

        public string IdKlijenta { get; set; }

        public DateTime PocetniDatum { get; set; }

        public DateTime KrajnjiDatum { get; set; }

        [Timestamp]
        public byte[] RowVersion { get; set; }
    }
}

File: PUSGSProjekat/src/app/entities/brza-rezervacija-vozila/brza-rezervacija-vozila.ts
import { NumberValueAccessor } from '@angular/forms';
import { Vozilo } from '../vozilo/vozilo';

export class BrzaRezervacijaVozila {
    id: number;
    idRentacar: number;
    idVozila: number;
    idKlijenta: string;
    pocetnaCena: number;
    novaCena: number;
    popust:number;
    pocetniDatum: string;
    krajnjiDatum:string;
    rowVersion:any[];

    constructor(id:number, idRentacar: number, idVozila: number, idKlijenta: string,popust: number,pocetnaCena: number, novaCena: number,pocetniDatum:string,krajnjiDatum:string, rowVersion:any[]) {
        this.id = id;
        this.idRentacar = idRentacar;
        this.idVozila = idVozila;
        this.idKlijenta = idKlijenta;
        this.pocetniDatum = pocetniDatum;
        this.krajnjiDatum = krajnjiDatum;
        this.pocetnaCena = pocetnaCena;
        this.novaCena = novaCena;
        this.popust = popust;
        this.rowVersion = rowVersion;
    }

}